scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1205,8 +1200,9 @@ private void updateStartOrEndOrScheduledtime(final long jobId, final long taskId
         runWithTransaction(new SessionWork<Void>() {
             @Override
             public Void executeWork(Session session) {
+
                 Query query = session.createQuery(
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use Hibernate's parameter binding instead of concatenation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2077)